### ISSUE TITLE
Test code showing on page

### ISSUE THREAD

**Issue by BrianaCamp** [FreeCodeCamp/FreeCodeCamp#2602]
Challenge http://www.freecodecamp.com/challenges/waypoint-comment-your-javascript-code has an issue. Please describe how to reproduce it, and include links to screenshots if possible.

Hi there, 
I came across this naturally from going through the jQuery exercises, I refreshed the page thinking what I imagine is test code to disappear, but alas, it's still there. This is what I'm seeing.
![screenshot 2015-08-23 13 14 01](https://cloud.githubusercontent.com/assets/7364028/9430008/35d1f962-4999-11e5-890d-7acf0e61b53d.png)

Hope this helps!
Thanks,
Bri

**Comment by benschenker** [comment]
Hey @BrianaCamp thanks for posting an issue!

That should be showing up there. It's part of all the javascript based challenges that you are able to see the tests that are running against your code. See #2231 for a similar issue/explanation.

This can be closed as a duplicate.  Take a look at the [Bug Posting Guidelines](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Help-I've-Found-a-Bug) for some tips to get your questions answered more quickly.

**Comment by BrianaCamp** [comment]
Ah so sorry! Thanks for the info, I'll definitely follow the guidelines
from now on!
When I attempted to look at the guidelines before submitting a request, I
realized on page :
https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md
The "Help! I've found a bug" link directs you to the wrong page. Looks like
the exclamation point should not be in the URL.

Hope this helps and thank you for your time! Sorry for the duplicate issue!

Bri
